HomeWorkSolution/Task2/Program.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task2
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {   
+            int num1 = InputNumber();
+            int num2 = InputNumber();
+            Console.WriteLine("Enter a math sign");
+            char sign = Console.ReadLine()[0]; //Read 1st symbol of an entered string
+
+            Console.WriteLine("Try to calculate it yourself!");
+            Console.WriteLine("Your answer is:");
+            int result = int.Parse(Console.ReadLine());          
```

Comment:
type.**Parse()** - great solution!
I would suggest to make a small investigation - take a look this method:  type.**TyParse()**
Decompile both methods and tell me which one needs to use.

Reply:
Ok i'll do my best :)